modules/ocf_irc/manifests/init.pp

```diff
@@ -1,4 +1,11 @@
 class ocf_irc {
   include ocf_ssl
+  include ocf_irc::ircd
+  include ocf_irc::services
   include ocf_irc::slack
+
+  # Make the irc user able to read the certs for running the IRCd with SSL
+  user { 'irc':
+    groups => 'ssl-cert',
+  }
```

Comment:
Thanks, forgot about that, that's a great idea. I think both the `inspircd` and `anope` packages add an irc user, but I'll just pick one since that should be enough to create the user properly.